.github/workflows/comment.yml

```diff
@@ -0,0 +1,59 @@
+name: comment
+on:
+  workflow_run:
+    workflows: ['build']
+    types: [completed]
+jobs:
+  pr_comment:
+    if: github.event.workflow_run.event == 'pull_request' && github.event.workflow_run.conclusion == 'success'
+    runs-on: ubuntu-latest
+    timeout-minutes: 35
+    steps:
+      - uses: actions/github-script@v6
+        with:
+          script: |
+            const { owner, repo } = context.repo;
+            const run_id = ${{ github.event.workflow_run.id }};
+            const pull_head_sha = '${{github.event.workflow_run.head_sha}}';
+
+            const issue_number = await(async () => {
+                const pulls = await github.rest.pulls.list({ owner, repo });
+                for await (const { data } of github.paginate.iterator(pulls)) {
+                    for (const pull of data) {
+                        if (pull.head.sha === pull_head_sha) {
+                            return pull.number;
+                        }
+                    }
+                }
+            })();
+            if (issue_number) {
+                core.info(`Using pull request ${issue_number}`);
+            } else {
+                return core.error(`No matching pull request found`);
+            }
+
+            const { data: { artifacts } } = await github.rest.actions.listWorkflowRunArtifacts({ owner, repo, run_id });
+            if (!artifacts.length) {
+                return core.error(`No artifacts found`);
+            }
+            let body = `Download the artifacts for this pull request:\n`;
+            let hidden_windows_artifacts = `\n\n <details><summary>Windows</summary>\n`;
+            for (const art of artifacts) {
+                if (art.name.includes('w64')) {
```

Comment:
is this applicable to our artifacts?
(did you test this workflow with your repo?)

Reply:
I think so. Or any suggestion about keywords?

https://github.com/natural-harmonia-gropius/mpv/pull/1#issuecomment-1565429147